requirements_test_pre_commit.txt

```diff
@@ -4,3 +4,4 @@ black==23.9.1
 codespell==2.2.2
 ruff==0.0.292
 yamllint==1.32.0
+numpy==1.26.0
```

Comment:
This is a requirement that comes in from core itself and should not be in here.

If you can't run it, it means you didn't install the core dev env correctly, or it means your editable install it outdated (which can be fixed by `pip install -e .`)

../Frenck